src/plugins/paginate/paginate.js

```diff
@@ -234,7 +274,7 @@ const componentName = "wb-paginate",
 	};
 
 // When a filter is updated
-$document.on( "click", "." + pagerClass + " a", function()  {
+$document.on( "click", "." + pagerClass + " a, ." + pagerClass + " button", function()  {
```

Comment:
Please add a scroll to the top of the pagination page when buttonsUI is activated.

Reply:
Fixed in commit "Made recommended changes".